ovh/resource_cloud_project_gateway_test.go

```diff
@@ -0,0 +1,78 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+var testAccCloudProjectGatewayConfig = `
```

Comment:
you should add a resource of type `ovh_vrack_cloudproject` first to make sure that the cloud project is linked to a vrack, you can do the same think as here: https://github.com/ovh/terraform-provider-ovh/pull/565/files

You will also need to add a `depends_on = [ovh_vrack_cloudproject.attach]` to resource `ovh_cloud_project_network_private.mypriv`

Reply:
i've made the change but i can't test anymore 
due to this 
` "OVHcloud API error (status code 409): \"Project is already on a private network\"`